tao_compiler/mlir/disc/transforms/lhlo_fusion.cc

```diff
@@ -543,7 +566,18 @@ struct DiscFusionPass : public DiscFusionPassBase<DiscFusionPass> {
   }
 
   bool ApplyFusionPlan(FusionPlan& plan) {
```

Comment:
`ApplyFusionPlan ` is called for each basic block. in case we have multiple blocks (e.g. control flow cases), 
move the initialization of `debug_max_fusion_numbers ` outside of this function may be better?

Reply:
TBD.